homeassistant/components/sky_remote/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""The Sky Remote Control integration."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+PLATFORMS = [Platform.REMOTE]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Sky remote."""
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
+    return True
```

Comment:
Is there a way we can check if we can connect to the device succesfully? If we can't we should raise `ConfigEntryNoyReady` so we try again later

Reply:
Added a best effort check that a successful connection takes place and if not raises ConfigEntryNotReady.